not correct

(| res |) -> Success(
    SubmissionDetail {
        status_code: 11,
        lang: "rust",
        run_success: true,
        status_runtime: "N/A",
        memory: 2016000,
        question_id: "1",
        elapsed_time: 72,
        compare_result: "000000000000000000000000000000000000000000000000000000000",
        code_output: "[1]",
        std_output: "",
        last_testcase: "[2,7,11,15]\n9",
        expected_output: "[0,1]",
        task_finish_time: 1691456840302,
        task_name: "judger.judgetask.Judge",
        finished: true,
        status_msg: "Wrong Answer",
        state: "SUCCESS",
        fast_submit: false,
        total_correct: Some(
            0,
        ),
        total_testcases: Some(
            57,
        ),
        submission_id: "454275439",
        runtime_percentile: None,
        status_memory: "N/A",
        memory_percentile: None,
        pretty_lang: "Rust",
    },
)

correct

(| res |) -> Success(
    SubmissionDetail {
        status_code: 10,
        lang: "rust",
        run_success: true,
        status_runtime: "4 ms",
        memory: 2216000,
        question_id: "1",
        elapsed_time: 79,
        compare_result: "111111111111111111111111111111111111111111111111111111111",
        code_output: "",
        std_output: "",
        last_testcase: "",
        expected_output: "",
        task_finish_time: 1691456946959,
        task_name: "judger.judgetask.Judge",
        finished: true,
        status_msg: "Accepted",
        state: "SUCCESS",
        fast_submit: false,
        total_correct: Some(
            57,
        ),
        total_testcases: Some(
            57,
        ),
        submission_id: "454275872",
        runtime_percentile: Some(
            50.7277,
        ),
        status_memory: "2.2 MB",
        memory_percentile: Some(
            78.7942,
        ),
        pretty_lang: "Rust",
    },
)


----


[crates/leetcode-api/tests/lc_manual.rs:42:5] &res = RunResult {
    elapsed_time: 0,
    finished: true,
    task_name: "judger.judgetask.Judge",
    status_code: 20,
    status_msg: "Compile Error",
    question_id: "100345",
    std_output: "",
    expected_output: "",
    last_testcase: "",
    code_answer: [],
    compare_result: "",
    correct_answer: false,
    expected_code_answer: [],
    expected_code_output: [],
    pretty_lang: "Rust",
    lang: "rust",
    memory: 0,
    status_memory: "N/A",
    memory_percentile: None,
    status_runtime: "N/A",
    runtime_percentile: None,
    run_success: false,
    state: "SUCCESS",
    std_output_list: [],
    submission_id: "535418998",
    task_finish_time: 1716882420090,
    total_correct: None,
    total_testcases: None,
    full_runtime_error: "",
    runtime_error: "",
    compile_error: "Line 46: Char 29: error: no function or associated item named `mechanical_accumulator` found for struct `Solution` in the current scop
e (solution.rs)",
    full_compile_error: "Line 46: Char 29: error: no function or associated item named `mechanical_accumulator` found for struct `Solution` in the current
 scope (solution.rs)\n   |\n31 | struct Solution;\n   | --------------- function or associated item `mechanical_accumulator` not found for this struct\n..
.\n46 |         let ret = Solution::mechanical_accumulator(param_1);\n   |                             ^^^^^^^^^^^^^^^^^^^^^^ function or associated item
not found in `Solution`\nFor more information about this error, try `rustc --explain E0599`.\nerror: could not compile `prog` (bin \"prog\") due to previo
us error",
}
